pkgs/development/python-modules/inflate64/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "inflate64";
+  version = "1.0.0";
+
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-MniCe4A88Aah3yUfPhM3TH0m23eeWjMynMEXibgEvC0=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
```

Comment:
Tests? (applies to most packages here — do note that most PyPI source releases don't include tests and so it might be better to find the original repository instead, but at the very least we should have pythonImportsCheck)

Reply:
Switched to using the upstream GH and Codeberg repos.

Reply:
Still need to add tests (e.g. via `nativeCheckInputs = [ pytestCheckHook ];`)